app/mailers/comment_notification_mailer.rb

```diff
@@ -1,22 +1,36 @@
 class CommentNotificationMailer < ApplicationMailer
-  def email_notification(comment)
+
+  def reviewer_notification(comment, reviewers)
     @comment = comment
     @proposal = comment.proposal
 
     # Email all reviewers of this proposal if notifications is true unless they made the comment
-    bcc = @proposal.ratings.map do |rating|
-      user = rating.user
-      if rating.event_teammate.try(:should_be_notified?) && @comment.user_id != user.id
-        user.email
-      end
+    to = reviewers.map do |reviewer|
+      reviewer.email if @comment.user_id != reviewer.id
     end.compact
 
-    if bcc.any?
+    if to.any?
       mail_markdown(
-                    bcc: bcc,
+                    to: to,
+                    from: @proposal.event.contact_email,
+                    subject: "#{@proposal.event.name} CFP: New comment on '#{@proposal.title}'")
+    end
+  end
+
+  def speaker_notification(proposal, comment, speaker_users)
+    @proposal = proposal
+    @comment = comment
+
+    to = speaker_users.map do |user|
```

Comment:
Like the reviewer mailer, you don't need to check who made the comment since they'll only be notified of reviewer activity.  This makes it a one liner following the same symbol.to_proc technique.

Also, speaker_users is an odd name since we used reviewers above (vs reviewer_users).  I'd just simplify to speakers for your method argument.